hm_pyhelper/lock_singleton.py

```diff
@@ -11,17 +11,30 @@
 class LockSingleton(object):
```

Comment:
It doesn't seem like `LockSingleton` is still a good name for this. It is now possible to create many different instances. I think `IpcLock` or `InterprocessLock` are more reflective of it now.

Reply:
`InterprocessLock` sounds good. :+1: